packages/ncids-css/scss/components/nci-header/internal/_search.scss

```diff
@@ -88,6 +95,26 @@
           max-width: units(6);
         }
       }
+      .search-button__cancel {
+        color:  color("primary");
```

Comment:
I don't know why this is needed here if the usa-button is the primary color. I can only assume it was from before when there was no usa-button. This should be fixed someday, but not needed for right now.